howto/android/graphics-debugging-with-renderdoc.md

```diff
@@ -0,0 +1,33 @@
+[Renderdoc](https://github.com/baldurk/renderdoc) is a graphics frame debugger which supports Android applications.
+
+Debugging graphics can be challenging and [Renderdoc](https://github.com/baldurk/renderdoc) is a great tool to provide insight into Vulkan and OpenGL command execution. The following describes how you can use Renderdoc with Anbox Cloud.
```

Comment:
Isn't the first sentence redundant with the second one?

Reply:
Yes, I will drop the first one.